build-system/common/check-package-manager.js

```diff
@@ -310,13 +310,19 @@ function runGulpChecks() {
 }
 
 function checkPythonVersion() {
-  // Python prints its version to stderr: https://bugs.python.org/issue18338
-  const pythonVersionResult = getStderr(`${pythonExecutable} --version`).trim();
+  // Python2 prints its version to stderr (fixed in Python 3.4)
+  // See: https://bugs.python.org/issue18338
+  const pythonVersionResult =
+    getStderr(`${pythonExecutable} --version`).trim() ||
+    getStdout(`${pythonExecutable} --version`).trim();
   const pythonVersion = pythonVersionResult.match(/Python (.*?)$/);
   if (pythonVersion && pythonVersion.length == 2) {
-    const recommendedVersion = '2.7';
+    // Python 2.7 is EOL but still supported
+    // Python 3.5+ are still supported (TODO: deprecate 3.5 on 2020-09-13)
+    // https://devguide.python.org/#status-of-python-branches
+    const recommendedVersion = /^2\.7|^3\.[5-9]/;
```

Comment:
I have three minor changes to suggest here:
1. `recommendedVersion` is used on line 337 to print the (range of) recommended version(s). Can you replace `cyan(recommendedVersion)` on that line with something like `cyan('2.7 or 3.5+')`?
2. `recommendedVersion` is used on line 341 to print a link to the download location. Can you replace `https://www.python.org/download/releases/${recommendedVersion}` on that line with `https://www.python.org/downloads`? (The download link no longer includes `releases/` in the url.)
3. For maintainability, can you rename the `recommendedVersion` variable on line 232 to `recommendedVersionRegex` or something similar, to make sure no one tries to use it as a raw version string in future?

Reply:
Thanks for catching the `recommendedVersion` issue I introduced. Fixed.